guides/v2.2/ui_comp_guide/components/ui-bookmarks.md

```diff
@@ -21,3 +21,21 @@ Option | Description | Type | Default Value
 Extends [`uiCollection`]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_uicollection_concept.html):
 
 -  [`app/code/Magento/Ui/view/base/web/js/grid/controls/bookmarks/bookmarks.js`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/js/grid/controls/bookmarks/bookmarks.js)
+-  [`app/code/Magento/Ui/view/base/web/templates/grid/controls/bookmarks/view.html`]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Ui/view/base/web/templates/grid/controls/bookmarks/view.html)
+
+## Example
```

Comment:
Please, use "Examples" title as we do for the most devdocs pages. Also, it would be nice to have a subheader that says something like "Add a bookmark to the listing toolbar"

See the following [example](https://devdocs.magento.com/guides/v2.3/ui_comp_guide/components/ui-fileuploader.html#example)